docs/self-managed/operational-guides/update-guide/840-to-850.md

```diff
@@ -0,0 +1,21 @@
+---
+id: 840-to-850
+title: Update 8.4 to 8.5
+description: "Review which adjustments must be made to migrate from Camunda 8.4.x to Camunda 8.5.0."
+---
+
+The following sections explain which adjustments must be made to migrate from Camunda 8.4.x to 8.5.x for each component.
+
+## Zeebe
+
+### Deprecated broker health checks
+
+The broker health check routes have moved, and the old routes are now deprecated. Specifically, the following routes will return [a status code of 301](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301) and redirect you. See the table below about the new mappings:
+
+| Old route                               | **New route**                                                 |
+| --------------------------------------- | ------------------------------------------------------------- |
+| http://{zeebe-broker-host}:9600/health  | **http://{zeebe-broker-host}:9600/actuator/health/status**    |
+| http://{zeebe-broker-host}:9600/ready   | **http://{zeebe-broker-host}:9600/actuator/health/readiness** |
+| http://{zeebe-broker-host}:9600/startup | **http://{zeebe-broker-host}:9600/actuator/health/startup**   |
+
+Please migrate to the new routes in your deployments. **If you're using the official Helm charts, then you don't have to do anything here.**
```

Comment:
:question: Shouldn't we update in the helm charts, even if the redirect works?

Reply:
hmm.. Could be tricky now that helm version is not the same as Zeebe version.

Reply:
Yes, there's an open issue to do this (and also one for the controller/operator) :+1: I'll coordinate with them about it